birdhouse/components/stac/config/proxy/conf.extra-directives.d/stac.conf.template

```diff
@@ -0,0 +1,7 @@
+    map $http_origin $stac_origin_allowed {
```

Comment:
Remind me what the Nginx `map` directive do again?

This block means if the `http_origin` variable exists, create `stac_origin_allowed` variable with value `${BIRDHOUSE_PROXY_SCHEME}://${BIRDHOUSE_FQDN_PUBLIC}; ${STAC_ADDITIONAL_CORS_ORIGINS}` ?